bwi_logging/src/bwi_logging/logger_node.py

```diff
@@ -70,17 +75,34 @@ def main(argv=None):
     logdir.chdir()                      # change to that directory
 
     # get the file prefix
-    prefix = rospy.get_param('~prefix', 'bwi')
+    prefix = rospy.get_param('~prefix', DEFAULT_PREFIX)
     rospy.loginfo('logging with prefix: ' + str(prefix))
 
     # this is the command we will issue:
     cmd = ['rosbag', 'record', '-o' + prefix, '-j']
     cmd.extend(topics.split())
     rospy.loginfo('running command: ' + str(cmd))
 
+    # run the rosbag command
+    status = subprocess.call(cmd)
+
+    rospy.loginfo('rosbag returned status: ' + str(status))
+
+    if status == 0 and prefix == DEFAULT_PREFIX:
+
+        rospy.loginfo('start uploading bags')
+
+        # In the background, begin uploading the newly-written bag to
+        # the BWI server.  The setsid isolates the uploading scripts
+        # from ROS shutdown signals.  The -w120 waits two minutes
+        # before starting to upload.
+        upload_cmd = ['/usr/bin/setsid', '/usr/local/bin/bwi',
+                      'bags', '-w120', '-d', logdir.pwd(), prefix, '&']
```

Comment:
Yes. Sometimes it works and sometimes it does not.

Given the fact that later runs will transfer earlier data, I figure that's better than what we currently have.